docs/platforms.rst

```diff
@@ -0,0 +1,51 @@
+.. _platforms:
+
+Platforms
+=========
+
+Bottery design always considered the possibility to integrate to any chat platform. At the code level, there're many abstractions to make it possible. Bottery works out of the box with two chat platforms, Facebook Messenger and Telegram but it give the tools to easly integrate to any specific platform.
+
+All of this platforms has different needs and different settings, but the views, when written without using specific platform features, should work for all of them.
```

Comment:
I think here that would be nice to "show off" that Bottery simplies particular structures and let you make reusable codes for both platforms. What do you think?

Reply:
suggestions?